reflex/compiler/utils.py

```diff
@@ -429,11 +430,14 @@ def add_meta(
     Returns:
         The component with the metadata added.
     """
-    meta_tags = [Meta.create(**item) for item in meta]
+    if len(meta):
+        print(meta[0])
 
-    children: list[Any] = [
-        Title.create(title),
+    meta_tags = [
+        item if isinstance(item, Link) else Meta.create(**item) for item in meta
```

Comment:
```suggestion
        item if isinstance(item, Component) else Meta.create(**item) for item in meta
```

Maybe this, so we have even more flexibility down the road without breaking current compatibility.